colabs/wandb_registry/zoo_wandb.ipynb

```diff
@@ -0,0 +1,984 @@
+{
```

Comment:
<p>-Sorry for the product marketing nitpickiness as there are tons of discrepancies across our docs and sites regarding capitalization and naming for the Registry feature. In this context, "Dataset" and "Model" should not be capitalized when discussing the dataset registry and the model registry, both of which are part of Registry.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/560/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>